.buildkite/pipeline.yml

```diff
@@ -225,6 +225,22 @@ steps:
         command: "julia --color=yes --project=examples examples/hybrid/driver.jl --vert_diff true --moist equil --rad clearsky --microphy 0M --job_id sphere_aquaplanet_rhoe_equilmoist_clearsky --dt 200secs --t_end 2.5days"
         artifact_paths: "sphere_aquaplanet_rhoe_equilmoist_clearsky/*"
 
+      - label: ":computer: held suarez (ρe_tot) topography"
+        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --forcing held_suarez --job_id sphere_held_suarez_rhoe_dcmip_topography --t_end 6days --topography DCMIP200 --dt 200secs --regression_test false --upwinding third_order --rayleigh_sponge true --viscous_sponge true" 
```

Comment:
do we have to use both sponges here?

Reply:
No; I added an item in buildkite to demonstrate that